### PR TITLE
Moved fargate example from Quickstart repo to CFN repo

### PR DESCRIPTION
**Pull request by SuperMohit**
## Description

Contains sample applications that are containerised for building an MVP using Atlas.

Link to any related issue(s): 

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation (if appropriate)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by Zuhairahmed — DISMISSED** [review]
LGTM

File: Quickstart examples/fargate/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+quickstart-mongodb-atlas-mean-stack-aws-fargate-integration—Quick Start
+For architectural details, step-by-step instructions, and customization options, see the deployment guide.
+
+To post feedback, submit feature ideas, or report bugs, use the Issues section of this GitHub repo.
+
+To submit code for this Quick Start, see the AWS Quick Start Contributor's Kit.
```
Comment (andreaangiolillo) [inline Quickstart examples/fargate/README.md:6]: - Can we add the other readme files in the client and server?
- Can we add the link to the step-by-step instruction?

**Review by andreaangiolillo — CHANGES_REQUESTED** [review]

File: Quickstart examples/fargate/README.md (hunk 1 above)
Comment (Zuhairahmed) [reply Quickstart examples/fargate/README.md:6]: @poornatejakonathampi @Adelmar92 ^^

Comment (poornatejakonathampi) [reply Quickstart examples/fargate/README.md]: Added build scripts for both client and server applications. Updated readme file.

**Comment by Zuhairahmed** [comment]
> I will wait the AWS team to approve the quickstart PR before approving this

@andreaangiolillo all outstanding PRs have been merged in by AWS for all Quick Starts

**Review by andreaangiolillo — DISMISSED** [review]

**Review by Zuhairahmed — APPROVED** [review]

**Review by andreaangiolillo — APPROVED** [review]
I was able to run the quickstart

**Pull request merged**